l10n_ar_account_tax_settlement/models/account_journal.py

```diff
@@ -1121,71 +1119,85 @@ def sicore_aplicado_files_values(self, move_lines):
             payment = line.payment_id
             pay_group = payment.payment_group_id
             move = line.move_id
-            if not payment:
-                raise ValidationError(_(
-                    'La liquidación de percepciones todavía no está '
-                    'implementada. Apunte contable: %s' % line.id))
 
-            # Codigo del Comprobante         [ 2]
-            content += (payment.payment_type == 'inbound' and '02') or (
-                payment.payment_type == 'outbound' and '06') or '00'
-
-            # Fecha Emision Comprobante      [10] (dd/mm/yyyy)
-            content += fields.Date.from_string(
-                pay_group.payment_date).strftime('%d/%m/%Y')
+            #si tengo payment es una retención, sino es una percepción y tengo que sacar la información de la factura (del move)
+            if payment:
+                # Codigo del Comprobante         [ 2]
+                content += (payment.payment_type == 'inbound' and '02') or (
+                    payment.payment_type == 'outbound' and '06') or '00'
 
-            # Numbero Comprobante            [16]
-            content += '%016d' % int(
-                re.sub('[^0-9]', '', move.l10n_latam_document_number))
+                # Fecha Emision Comprobante      [10] (dd/mm/yyyy)
+                content += fields.Date.from_string(
+                    pay_group.payment_date).strftime('%d/%m/%Y')
+                # Numero Comprobante            [16]
+                content += '%016d' % int(re.sub('[^0-9]', '', move.l10n_latam_document_number))
+                #Importe del comprobante
+                codop = '1'
+                issue_date = payment.payment_date
+                amount_tot = abs(payment.payment_group_id.payments_amount)
```

Comment:
why you applied abs() here? I ask because the preivuos version that do not have retention was without the abs
```suggestion
                amount_tot = abs(payment.payment_group_id.payments_amount)
```